backend/src/main/java/ca/gov/dtsstn/cdcp/api/service/domain/ConfirmationCode.java

```diff
@@ -0,0 +1,19 @@
+package ca.gov.dtsstn.cdcp.api.service.domain;
+import org.immutables.value.Value.Immutable;
+
+import java.time.Instant;
+
+@Immutable
+public interface  ConfirmationCode extends BaseDomainObject {
```

Comment:
Extra space here

Reply:
should be fixed with the formatter ive used